src/main/java/com/google/cloud/tools/gradle/appengine/standard/AppEngineStandardPlugin.java

```diff
@@ -144,7 +138,7 @@ private void configureExtensions() {
           if (Strings.isNullOrEmpty(runExtension.getProjectId())) {
             runExtension.setProjectId(deploy.getProjectId());
           }
-          runExtension.setDeployTargetResolver(standardResolver);
+          runExtension.setDeployTargetResolver(deploy.getDeployTargetResolver());
```

Comment:
Will `deploy.getDeployTargetResolver()` return null? A new `deploy` instance is created in this method, and I don't immediately see anyone calling `setDeployTargetResolver()` on `deploy`.

Reply:
Yeah, that's where the code is a little wonky. It might worth just instantiating a new one for clarification. But, it is created in another area where we create the Deploy Extension: https://github.com/GoogleCloudPlatform/app-gradle-plugin/pull/325/files#diff-61ff2d5be7163076a4c1810bd162f73eR129

I think I might just create a new object here.